modules/claims_api/app/sidekiq/claims_api/claim_establisher.rb

```diff
@@ -44,12 +44,12 @@ def perform(auto_claim_id) # rubocop:disable Metrics/MethodLength
       queue_special_issues_updater(auto_claim.special_issues, auto_claim)
     rescue ::Common::Exceptions::BackendServiceException => e
       auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED
-      auto_claim.evss_response = get_error_message(e) # e.original_body
+      auto_claim.evss_response = get_error_message(e)
       auto_claim.form_data = orig_form_data
       auto_claim.save
     rescue => e
       auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED
-      auto_claim.evss_response = get_error_message(e) # (e.errors.presence || e.detailed_message)
+      auto_claim.evss_response = get_error_message(e)
       auto_claim.form_data = orig_form_data
       auto_claim.save
```

Comment:
Cleanup, removes comments

Reply:
Thank you for cleaning that up, I always had a 'doh' moment every time I saw that in there.